app.json

```diff
@@ -76,6 +76,16 @@
       "description": "Set to 'twitter' or 'facebook', or 'all' to disable any login method other than email.",
       "required": "false",
       "value": ""
+    },
+    "ELECTION_DATE": {
+      "description": "date of the election in the format YYYY-MM-DD",
+      "required": "false",
+      "value": ""
+    },
+    "ELECTION_TYPE": {
+      "description": "Type of election, general or by / g or b",
```

Comment:
I think it should just be `general` or `by-election`.  It's too easy to make a typo between `g` and `b` (the keys are right next to each other) so requiring the full word introduces some error-correction bits.  Additionally, it helps others if we avoid somewhat cryptic values / identifiers.